crates/bevy_render/src/texture/texture_attachment.rs

```diff
@@ -120,3 +120,41 @@ impl DepthAttachment {
         }
     }
 }
+
+/// A wrapper for a [`TextureView`] that is used as a [`RenderPassColorAttachment`] for a view
+/// targets final output texture.
```

Comment:
This sentence has 2 verbs, no?

Reply:
I don't think so, targets isn't a verb here it's "view targets" it might be missing an apostrophe, but my english isn't great so I can't comment on that.

Reply:
It's missing an apostrophe, thanks!